MonoGame.Framework/Matrix.cs

```diff
@@ -7,13 +7,35 @@
 using System.Runtime.Serialization;
 
 namespace Microsoft.Xna.Framework
-{
+{ 
+    /// <summary>
+    /// Represents the right-handed 4x4 float-based matrix, which can store translation,scale and rotation information.
+    /// </summary>
     [DataContract]
     [DebuggerDisplay("{DebugDisplayString,nq}")]
     public struct Matrix : IEquatable<Matrix>
     {
         #region Public Constructors
-        
+
+        /// <summary>
+        /// Creates a new instance of <see cref="Matrix"/> struct.

```

Comment:
We don't need to reference `Matrix` within the `Matrix`.   They are already reading the matrix docs.

Just say:  `Constructs a new matrix.`.